### PR TITLE
LB-686 Cleaned up a variety of user information pages

### PR DESCRIPTION
**Pull request by tomnieuwland**
# Problem

https://tickets.metabrainz.org/browse/LB-686
Separation of importing information and scrobbling information. I also took it upon myself to condense some information into a single source of truth, rather than having the same thing written differently across a few pages.

# Solution

- Changed .gitignore to include .idea files from Intellij based IDEs
- Added add-data.html for instructions on adding listens to Listenbrainz
- Added import-data.html for instructions on importing data form existing third parties
- Moved Last.fm importing information from contribute.html and moved it to add-data.html
- Moved contribution information from index.html and moved it to contribute.html
- Moved and reworded Spotify "import" information from import.html to add-data.html
- Removed Import tab in favor of `Data > Import Data`
- Re-arranged dropdowns to be a little less of a mess
- Deleted api-docs.html and replaced its references with a link to the readthedocs url (page was redundant)

# Action

Potentially suggest rewording of some text? Things may be outdated, some of this stuff was last touched in 2015 according to git blame

### CODE REVIEW

**Comment by shivam-kapila** [comment]
@brainzbot add to whitelist

**Comment by shivam-kapila** [comment]
@brainzbot retest this please

File: listenbrainz/webserver/templates/index/import-data.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{%- extends 'base.html' -%}
+{%- block title -%}Adding Data - ListenBrainz{%- endblock -%}
```
Comment (paramsingh) [inline listenbrainz/webserver/templates/index/import-data.html:2]:
```suggestion
{%- block title -%}Importing Data - ListenBrainz{%- endblock -%}
```

File: listenbrainz/webserver/templates/navbar.html [hunk 2]
```diff
@@ -24,30 +24,30 @@
...
           <a class="dropdown-toggle" data-toggle="dropdown" href="#">
-             Contribute<span class="caret"></span>
+             Data<span class="caret"></span>
           </a>
           <ul class="dropdown-menu" role="menu">
-            <li><a href="{{ url_for('index.contribute') }}">Contribute</a></li>
-            <li><a href="{{ url_for('index.data') }}">Data</a></li>
-            <li><a href="{{ url_for('index.api_docs') }}">API Docs</a></li>
-            <li><a href="{{ url_for('index.roadmap') }}">Roadmap</a></li>
+            <li><a href="{{ url_for('index.add_data_info') }}">Adding data</a></li>
```
Comment (paramsingh) [inline listenbrainz/webserver/templates/navbar.html:48]: I think the adding data page should be in the top level instead of a dropdown, maybe with a better title? "Track your listens" or something that's more easily understandable for users?

**Review by paramsingh — CHANGES_REQUESTED** [review]
Thanks for doing this! Looks great to me, just had the two small comments

File: listenbrainz/webserver/templates/index/index.html [hunk 3]
```diff
@@ -22,7 +22,7 @@ <h2 class="page-title">Welcome to ListenBrainz!</h2>
         account, you can <a href="https://musicbrainz.org/register">create one</a>! Once you have an
         account, you can <a href="{{ url_for('profile.import_data') }}">import</a>
         data from Last.FM, <a href="{{ url_for('profile.import_data') }}">connect your Spotify account</a>,
```
Comment (paramsingh) [inline listenbrainz/webserver/templates/index/index.html:24]:
```suggestion
        data from Last.FM, <a href="{{ url_for('profile.connect_spotify') }}">connect your Spotify account</a>,
```

This is unrelated, but the spotify import link should go to the spotify page, now that the import page doesn't mention spotify.

**Review by paramsingh — CHANGES_REQUESTED** [review]

File: listenbrainz/webserver/templates/navbar.html (hunk 2 above)
Comment (tomnieuwland) [reply listenbrainz/webserver/templates/navbar.html:48]: @paramsingh Apologies on disappearing on this one, I had some family issues come up. I'll get this sorted some time in the next day or two!

**Review by paramsingh — APPROVED** [review]
